crates/sui-framework/packages/deepbook/sources/custodian.move

```diff
@@ -37,6 +37,9 @@ module deepbook::custodian {
         custodian: &Custodian<Asset>,
         user: ID
     ): (u64, u64) {
+        if (!table::contains(&custodian.account_balances, user)) {
```

Comment:
Could you add a comment explaining that this never aborts, and returns `(0, 0)` if `ID` does not exist in `custodian`?

Reply:
sure thing